### PR TITLE
`managed-clonerefs`: Multiarchitecture build

### PR DESCRIPTION
**Pull request by danilo-gemoli**
Add multiarch support to `managed-clonerefs`.

/cc @droslean

### CODE REVIEW

File: clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml [hunk 1]
```diff
@@ -7,44 +7,44 @@ spec:
...
     local: true
 ---
-apiVersion: build.openshift.io/v1
-kind: BuildConfig
+apiVersion: ci.openshift.io/v1
+kind: MultiArchBuildConfig
+
 metadata:
-  name: manage-clonerefs-arm64
+  name: manage-clonerefs
```
Comment (danilo-gemoli) [inline clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml:14]: Isn't `managed-clonerefs` the proper name?

Comment (droslean) [reply clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml:14]: Yes, but this change won't make any difference. We will build it only once. We need to implement a mechanism to build the image on demand. I guess we can follow the strategy build or add a cron configuration.

Comment (danilo-gemoli) [reply clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml:14]: Maybe something similar to `oc start-build` in a periodic?
Either way before start working on that, I wanted to make sure the build and mirror mechanism works as expected and we don't face any issue using the new image.

Comment (droslean) [reply clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml:14]: We can't use the `oc start-build`. This command is not related to the resource.

Comment (danilo-gemoli) [reply clusters/build-clusters/multi01/supplemental-ci-images/manage-clonerefs.yaml:14]: I mean developing something similar to that command.

File: clusters/build-clusters/multi01/supplemental-ci-images/managed-clonerefs.yaml [hunk 2]
```diff
@@ -7,44 +7,43 @@ spec:
...
+        - destinationDir: .
+          sourcePath: /ko-app/clonerefs
+      type: Dockerfile
+    strategy:
+      dockerStrategy:
+        imageOptimizationPolicy: SkipLayers
+        noCache: true
+      type: Docker
+  external_registries:
+  - registry.ci.openshift.org/ci/managed-clonerefs:latest
```
Comment (droslean) [inline clusters/build-clusters/multi01/supplemental-ci-images/managed-clonerefs.yaml:46]:
```suggestion
  - registry.ci.openshift.org
```

**Review by droslean — APPROVED** [review]

**Pull request merged**